src/Tools/dotnet-watch/test/AwaitableProcess.cs

```diff
@@ -134,30 +137,50 @@ public async Task<IList<string>> GetAllOutputLinesAsync(CancellationToken cancel
         private void OnData(object sender, DataReceivedEventArgs args)
         {
             var line = args.Data ?? string.Empty;
-            _logger.WriteLine($"{DateTime.Now}: post: '{line}'");
+
+            WriteTestOutput($"{DateTime.Now}: post: '{line}'");
             _source.Post(line);
```

Comment:
Could this ever throw after it's completed?

Reply:
Tried this on and it does not.